packages/babel-plugin-transform-classes/src/vanilla.js

```diff
@@ -522,9 +522,18 @@ export default class ClassTransformer {
       if (this._processMethod(node, scope)) return;
     }
 
+    // Check if it is a static get or set
+    if (node.static && (node.kind === "get" || node.kind === "set")) {
+      if (this._processSetGet(node, path)) return;
+    }
+
     this.pushToMap(node, false, null, scope);
   }
 
+  _processSetGet() {
```

Comment:
this seems a little bit clunky, but i guess it is somewhat forced upon because of the hierarchical class structure, I will file a refactoring request issue for this

Reply:
Agreed. I don't like that structure so much. Composition over inheritance they say.

Reply:
U can upvote this then - https://github.com/babel/babel/issues/7202 😄

Reply:
can't upvote more